### PR TITLE
fix wrong menu-selected

### LINKED ISSUE #876: Incorrect menu highlight in the homepage
Menu entry "topoguide" should not be hilighted in the homepage:
capture d ecran 2016-11-07 a 21 48 58

### PR DESCRIPTION
**Pull request by ginold**
fixes #876

i'm not sure what happened, but i just moved the code around and it fixed it :confused:

### CODE REVIEW

File: c2corg_ui/static/js/maincontroller.js [hunk 1]
```diff
@@ -78,9 +78,9 @@ app.MainController.prototype.isPath = function(path) {
     return 'home';
   // if topoguide, it can be all kinds of documents
   } else if (path === 'topoguide') {
-    return location.indexOf('waypoints') > -1 || location.indexOf('routes') > -1 || location.indexOf('outings') > -1
-            || location.indexOf('areas') > -1 || location.indexOf('images') > -1 || location.indexOf('books')
```
Comment (asaunier) [inline c2corg_ui/static/js/maincontroller.js]: Was this ``location.indexOf('books')`` (missing ``> -1``) the reason of the problem?
I am a bit in doubt that changing the order of the tests actually fixes it?!

Comment (ginold) [inline c2corg_ui/static/js/maincontroller.js]: oh maybe, I didn't spot the missing -1 !

I really don't find those lines too long. I mean...look at our html templates...:P

Comment (asaunier) [inline c2corg_ui/static/js/maincontroller.js]: The template lines ARE too long!

Comment (ginold) [inline c2corg_ui/static/js/maincontroller.js]: oh yeah they are! next to them, those are little tiny perfect lines.
Anyway - ok for merging?

Comment (asaunier) [inline c2corg_ui/static/js/maincontroller.js]: Changing the tests order is not relevant to me when the only required change was to add the missing ``> -1``.
I admit it is probably peanuts but it's better to test the most usual document first (waypoints, routes, outings). Book pages are more rarely used.

Comment (ginold) [inline c2corg_ui/static/js/maincontroller.js]: yep you're right. I will change it and merge afterwards.

Comment (ginold) [inline c2corg_ui/static/js/maincontroller.js]: I tried with another, probably clearer version : 

```
    var paths = ['/outings', '/waypoints', '/routes', '/images', '/areas', '/books'];
    return $.inArray(location, paths) > -1;
```

do you find it ok?

**Pull request merged**